Opcodes/emugens/.gitignore

```diff
@@ -0,0 +1 @@
+compile_commands.json
```

Comment:
I don't think this file belongs in the repo. This may be added to the base gitignore to ignore any compile-commands